cav_msgs/msg/ControlMessage.msg

```diff
@@ -0,0 +1,87 @@
+#
+# ControlMessage.msg
+#
+# Part of the CARMA Cloud geo-fence specification not part of the official SAE j2735 standard
+#
+# J2735 2016 message format.
+#
+# @author  Misheel Bayartsengel 
+# @version 0.1
+#
+# CARMA Cloud generates control messages based on geocoordinates as of a certain date-timestamp. 
+# CARMA Platform interprets the control messages to its own map context, which is consistent with 
+# maps being used by vehicles.
+# Transportation controls in the physical view and control messages in the information view convey 
+# necessary information to affect human and ADS behavior: vehicle type, when a control is active, 
+# area of effect, and constraints.
+
+# Version ::= IA5String (SIZE(36))
+string version 
+
+# Id ::= OCTET STRING (SIZE(16))
+uint8[16] id 
+
+# Updated time
+#  -- time when this message is last updated
+time updated
+
+# vTypes VehicleTypes ::= SET OF VehicleType
+j2735_msgs/VType[] vtypes
+
+# schedule Schedule
+cav_msgs/Schedule schedule
+
+# Regulatory ::= BOOLEAN
+bool regulatory
+
+# controlType ControlType ::= ENUMERATED
+j2735_msgs/ControlType control_type
+
+# controlValue ControlValue OPTIONAL ::= EXPLICIT CHOICE 
+j2735_msgs/ControlValue control_value
+bool control_value_exists
+
+# PathParts ::= INTEGER (0..127)
+uint8 path_parts
+uint8 PATH_PART_MAX = 127
+
+# Proj ::= IA5String (SIZE(0..255))
+string proj
+
+# Datum ::= IA5String (SIZE(0..50))
+string datum
+
+# Time of control
+time time
+
+# Longitude in degrees
+float64 longitude
+
+float64 LONGITUDE_MAX = 180.0
+float64 LONGITUDE_MIN = -179.9999999
+
+# Latitude in degrees
+float64 latitude
+
+float64 LATITUDE_MAX = 90.0
+float64 LATITUDE_MIN = -90.0
```

Comment:
I think we can drop these min/max variables here and just have a comment describing the range of lat/lon. Direct floating point comparison is rarely valid so these couldn't be used in a meaningful way.